<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientLobby</class>
 <widget class="QMainWindow" name="ClientLobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientLobby</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>JAZZ JACK RABBIT 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCreateMatch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a match</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCreateMatch_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Join a match</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCreateMatchCode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>4 digit code</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtJoinMatchCode">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>4 digit code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJoin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>available matches</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtAvailableMatches">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
